### DISCUSSION TITLE
PIP-14: Client agent standard

### DISCUSSION THREAD

**Discussion by ja7ad** [pactus-project/PIPs#53] [PIPs]
PIP-14: [Client agent standard](https://pips.pactus.org/PIPs/pip-14)

**Comment by b00f** [comment]
@Ja7ad 

To get the agent application name in Golang we can use this function:
```go
fmt.Println(filepath.Base(os.Args[0]))
```

**Reply by ja7ad** [reply]
@b00f Sure
